docs/rust-client/counter_contract_tutorial.md

```diff
@@ -360,16 +372,12 @@ Paste the following code at the end of your `src/main.rs` file:
 // -------------------------------------------------------------------------
 println!("\n[STEP 2] Call Counter Contract With Script");
 
-// Grab the first procedure hash
-let procedure_2_hash = procedures_vec[0].to_hex();
-let procedure_call = format!("{}", procedure_2_hash);
-
 // Load the MASM script referencing the increment procedure
 let file_path = Path::new("./masm/scripts/counter_script.masm");
 let original_code = fs::read_to_string(file_path).unwrap();
 
 // Replace the placeholder with the actual procedure call
-let replaced_code = original_code.replace("{increment_count}", &procedure_call);
+let replaced_code = original_code.replace("{increment_count}", &increment_count_root);
```

Comment:
Should we rename that variable to `increment_count_hash`? Just to make it consistent?